This patch updates code to move pine.conf from /usr/local/lib to /etc.
--- a/pith/filter.c
+++ b/pith/filter.c
@@ -7672,7 +7672,7 @@
 		}
 		else if(!strcmp(s, "PINE_CONF_PATH")){
 #if defined(_WINDOWS) || !defined(SYSTEM_PINERC)
-		    p = "/usr/local/lib/pine.conf";
+		    p = "/etc/pine.conf";
 #else
 		    p = SYSTEM_PINERC;
 #endif
@@ -7681,7 +7681,7 @@
 #ifdef SYSTEM_PINERC_FIXED
 		    p = SYSTEM_PINERC_FIXED;
 #else
-		    p = "/usr/local/lib/pine.conf.fixed";
+		    p = "/etc/pine.conf.fixed";
 #endif
 		}
 		else if(!strcmp(s, "PINE_INFO_PATH")){
